action/protocol/staking/handler_candidate_selfstake_test.go

```diff
@@ -258,7 +264,7 @@ func TestProtocol_HandleCandidateSelfStake(t *testing.T) {
 		},
 		{
 			"bucket is already selfstaked",
-			[]uint64{0, 3},
+			[]uint64{0, 10},
```

Comment:
why need this change

Reply:
the original test data is inaccurate, the candidate of a selfstake bucket should be equal to the owner